### ISSUE TITLE
Composer isn't obtaining the latest version of CCXT

### ISSUE THREAD

**Issue by rubobaquero** [ccxt/ccxt#546]
I have something like this in my `composer.json`:

```
{ 
  "require": {
      "ccxt/ccxt": "dev-master",
      [OTHER PACKAGES]
  },
  "minimum-stability": "dev",
  "prefer-stable": true
}
```

If i try to update CCXT to the last version running `composer update` I always get 1.10.68 version, while current is 1.10.72 version.

I have tried too to replace `"ccxt/ccxt": "dev-master"` by `"ccxt/ccxt": "*"` wihout any changes.

Am I doing something wrong?

**Comment by kroitor** [comment]
@rubobaquero i've just set up proper GitHub hooks, so now it should show current up to date version 1.10.72+.

**Comment by kroitor** [comment]
Strange things going on with it tho...

<img width="1127" alt="screen shot 2017-11-12 at 19 50 48" src="https://user-images.githubusercontent.com/1294454/32701233-dd473c1a-c7e2-11e7-9cf2-e9f4754c42a1.png">

**Comment by rubobaquero** [comment]
Latest version available now is 1.10.72 :-) I suppose that it can take a few minutes after version 1.10.73 is "distributed" to Packagist.

**Comment by kroitor** [comment]
Looks like packagist is updating the version, but does not update the package itself... This is the cause of the exmo fix not getting through. I'll update it manually.
